docusaurus/docs/dev-docs/preview.md

```diff
@@ -264,15 +268,100 @@ export async function GET(request: Request) {
 }
 ```
 
-### 5. Allow the front-end to be embedded
+### 5. [Front end] Allow the front-end to be embedded {#5-allow-frontend-embed}
 
 On the Strapi side, [the `allowedOrigins` configuration parameter](#allowed-origins) allows the admin panel to load the front-end window in an iframe. But allowing the embedding works both ways, so on the front-end side, you also need to allow the window to be embedded in Strapi's admin panel.
 
 This requires the front-end application to have its own header directive, the CSP `frame-ancestors` directive. Setting this directive up depends on how your website is built. For instance, setting this up in Next.js requires a middleware configuration (see [Next.js docs](https://nextjs.org/docs/app/building-your-application/configuring/content-security-policy)).
 
+### 6. [Front end] Detect changes in Strapi and refresh the front-end {#6-refresh-frontend}
+
+Strapi emits a `strapiUpdate` message to inform the front end that data has changed. 
+
+To track this, within your front-end application, add an event listener to listen to events posted through [the `postMessage()` API](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage). The listener needs to filter through messages and react only to Strapi-initiated messages, then refresh the iframe.
```

Comment:
>add an event listener to listen to events posted through [the `postMessage()` API]

Maybe it's overly specific but we could specify "posted _on the window object_"

Reply:
>then refresh the iframe

This seems to imply a hard refresh, maybe say "then refresh the iframe content"?